src/pretix/static/pretixbase/js/asynctask.js

```diff
@@ -152,6 +153,7 @@ function async_task_error(jqXHR, textStatus, errorThrown) {
 
             if (respdom.filter('#page-wrapper') && $('#page-wrapper').length) {
                 $("#page-wrapper").html(respdom.find("#page-wrapper").html());
+                setup_basics($("body"));
```

Comment:
I think this needs to be constrained to `#page-wrapper` in this case, otherwise existing elements on the body might be reset (e.g. collapsed-state).

```suggestion
                setup_basics($("#page-wrapper"));
```